### PR TITLE
installed python 2.7.13 using pyenv

### PR DESCRIPTION
**Pull request by kzidane**

### CODE REVIEW

File: update50 [hunk 1]
```diff
@@ -116,8 +116,9 @@ fi
 
 echo "FYI: installing Python may take a while. Thank you for your patience! <3"
 sudo chown -R ubuntu:ubuntu /opt/pyenv
+/opt/pyenv/bin/pyenv install --skip-existing 2.7.13
```
Comment (dmalan) [inline update50]: Is this going to take a while to install?

Comment (kzidane) [reply update50]: Takes about 3-4 minutes online to install Python 2.7.

Comment (dmalan) [reply update50]: Do we definitely benefit in the short term from this? Or should we wait until v4?

Comment (kzidane) [reply update50]: Don't think there's benefit for this in the short term.

**Pull request closed without merging**